src/components/footer/footer.tsx

```diff
@@ -0,0 +1,73 @@
+import { Divider } from '../cards/divider/divider';
+import { Label } from '../typography/label/label';
+import { FooterNav } from './footer-nav';
+import './footer.css';
+import company from './company-.svg';
+
+export function Footer() {
+  return (
+    <footer>
+      <div
+        style={{
+          display: 'flex',
+        }}
+      >
+        <div className="company-info">
+          <div>
+            <img src={company} />
+          </div>
+        </div>
```

Comment:
nao precisa dessa div de dentro ne?

```suggestion
        <div className="company-info">
          <img src={company} />
        </div>
```

Reply:
se não tiver essa div a img fica muito grande